app.js

```diff
@@ -58,6 +62,25 @@ const init = async () => {
     res.json(version);
   });
 
+  // Our specification download service can't use GraphQL because GraphQL specification requires the content-type
+  // that it returns be json, and we want to be able to return other content types, such as tab-separated-values,
+  // so that the user is automatically prompted to save the file from their browser.
+  const apiRoot = urlJoin(CLINICAL_SERVICE_ROOT, SUBMISSION_TEMPLATE_PATH);
+  app.get('/downloads', async (req,res) => {
```

Comment:
In order to keep route implementations separate from the main app/gql declaration, can we do the following:

- create a new directory called `routes`, with a file called `clinical.js`
- in `routes/clinical.js` lets create an [express.Router](http://expressjs.com/en/guide/routing.html#express-router) for clincialRoutes and define the implementation as `/template`. export this router.
- import the router in app.js and have it handle the path `/clinical`

End goal to resolve `/clinical/template` to the download code you wrote.